app/src/main/res/values/styles.xml

```diff
@@ -1,15 +1,21 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:windowContentTransitions" tools:targetApi="lollipop">true</item>
```

Comment:
Are you sure this won't crash below Lollipop? [Docs](https://developer.android.com/reference/android/annotation/TargetApi) say 
>Indicates that Lint should treat this type as targeting a given API level, no matter what the project target is.

So it basically only silences Lint.